ansible/testbed-cli.sh

```diff
@@ -343,6 +343,14 @@ function remove_topo
   echo Done
 }
 
+function redeploy_topo()
+{
+    remove_topo $@
```

Comment:
How about `remove_topo $@ || true`?
I noticed that sometimes `remove_topo` may fail. Then just run `add_topo` is able to redeploy the testbed.

Reply:
Sure. Changed.